### PR TITLE
BTRX - 620 - Convert about page

### PR DESCRIPTION
**Pull request by LeoForconesi**
## Addresses
[BTRX-620](https://broadinstitute.atlassian.net/browse/BTRX-620) : Convert About Page

## Changes
- Create react About page
- Add Router redirection

## Testing
- Click on the About link on the top navigation option, it should render the new page.
---

- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Get a thumb from Belatrix
- [ ] **Submitter**: Get a thumb from @rushtong
- [ ] **Submitter**: Merge to develop using github's "Squash and Merge" feature
- [ ] **Submitter**: Delete branch after merge

### CODE REVIEW

File: src/main/webapp/aboutPage/AboutPage.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+class AboutPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+
+  componentDidMount() {
+    console.log("ABOUT REACT")
```
Comment (rushtong) [inline src/main/webapp/aboutPage/AboutPage.js]: rm

File: src/main/webapp/aboutPage/AboutPage.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+body {
+    margin-top: 0;
+    padding-top: 70px;
+}
```
Comment (rushtong) [inline src/main/webapp/aboutPage/AboutPage.css]: Could this be in-lined on the main div?

File: src/main/webapp/aboutPage/AboutPage.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        ]),
+        p({ style: { fontFamily : styles.fontFamily, fontSize: styles.textFontSize }}, [
+          "The Broad's Office of Research Subject Protection (ORSP) is committed to helping our community adhere to " +
+          "federal regulations and institutional policies governing the protection of human subjects who make our" +
+          "research possible. To fulfill that mission, the Broad must ensure appropriate regulatory oversight and" +
+          "reliable documentation storage.  The ORSP Portal is an online platform where Broad staff members can upload " +
+          "documents for ORSP review (either as stand-alone submissions, or in preparation for an IRB protocol applications), " +
+          "store consent forms and regulatory approvals, and search for data use restrictions. Please contact ",
+          a({
+            href:"mailto:orsp-portal@broadinstitute.org"}, ["orsp-portal@broadinstitute.org"]
```
Comment (rushtong) [inline src/main/webapp/aboutPage/AboutPage.js:36]: Might be nice to introduce a library like https://github.com/jasonbellamy/react-mailto here. Totally optional - don't hold up the PR on this.

File: webpack.config.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ module.exports = {
     linkWizard: './src/main/webapp/linkWizard/index.js',
     mainIndex: './src/main/webapp/main/main_index.js',
     project: './src/main/webapp/project/index.js',
-    search: './src/main/webapp/search/index.js',
+    search: './src/main/webapp/search/index.js'
```
Comment (rushtong) [inline webpack.config.js]: nit: suggest that you revert this change.

File: grails-app/controllers/org/broadinstitute/orsp/IndexController.groovy [hunk 5]
```diff
@@ -8,7 +8,9 @@ class IndexController {
         render(view: "/index")
     }
 
-    def about() {}
+    def about() {
+        render(view: "/mainContainer/index")
+    }
```
Comment (rushtong) [inline grails-app/controllers/org/broadinstitute/orsp/IndexController.groovy:13]: I assume that we don't have the top nav bar converted into a react component at this time. Not sure where that is in the pipeline of work. Once these are integrated into `Main.js`, there should not be any calls to this method.

**Review by rushtong — APPROVED** [review]
👍 See minor comments inline.

**Review by dags — APPROVED** [review]

File: grails-app/controllers/org/broadinstitute/orsp/IndexController.groovy (hunk 5 above)
Comment (LeoForconesi) [reply grails-app/controllers/org/broadinstitute/orsp/IndexController.groovy:13]: That's correct, at the moment we are converting the remaining pages to React so we can later proceed to this final task, which is to get rid of grails dependency. Including redirections and model constants. This definitely will disappear.

File: src/main/webapp/aboutPage/AboutPage.js (hunk 3 above)
Comment (LeoForconesi) [reply src/main/webapp/aboutPage/AboutPage.js:36]: I've give a try on this, but I'm getting a TypeError on a what it seems to be a required node, i didn't keep trying to make it work, i'm suspecting that it might have to do with its old libraries usage.
I'll get back to this on a different PR.

File: grails-app/views/layouts/main_component.gsp [hunk 6]
```diff
@@ -111,7 +111,7 @@
 
     <g:layoutHead/>
 </head>
-<body>
+<body style="margin-top: 0; padding-top: 70px;">
```
Comment (LeoForconesi) [inline grails-app/views/layouts/main_component.gsp:114]: > Could this be in-lined on the main div?

@rushtong Is this acceptable? Moving this style to its div isn't taking any effect. The other option it comes to my head is to use a class...

Comment (rushtong) [reply grails-app/views/layouts/main_component.gsp:114]: Yes 👍
